### DISCUSSION TITLE
Best way to solve "Pull request with merge commit are not supported" and question

### DISCUSSION THREAD

**Discussion by qneill-sifive** [Mergifyio/mergify#5063] [Workflows]
We got "Pull request with merge commit are not supported" on one of our backport requests, what's the right way to solve this (using mergify preferably)?

And question: that seems like bad grammar - is it possible someone has hijacked or hacked our mergify account? (The bad grammar being typically one would prefix the non-distinct singular noun "merge commit" with "a", or make it plural, so either "with a merge commit" or "with merge commits" would be better).

**Comment by jd** [comment]
Hi @qneill-sifive !

Do not overestimate Mergify engineer's English level. The typo is legit and present in our source code, we just fixed it. Thank you for reporting! 😄 

As for the original question, there is no way to solve this with Mergify. This happens when the git operation is too complex to be done automatically, and more context is needed to know which branch of the tree to follow.

**Reply by qneill-sifive** [reply]
Okay thanks for the typo report.  I almost didn't report it because it was so minor.  Any little English shortcomings are overshadowed by the Mergify engineer's Git-Foo-Level and so are 100% forgivable!

As for this particular report - after exploring the shape and size of this branch ... I know why mergify gave up on it, it was a complex branch with at least 5 merge commits.

Thanks for helping.
